### DISCUSSION TITLE
layout problem

### DISCUSSION THREAD

**Discussion by NazarAli** [filamentphp/filament#8330] [Help] [answered]
i have problem in layout 
first one the select page size in pagination in dark mode not style correctly 

![1222](https://github.com/filamentphp/filament/assets/7056487/807a677c-f092-410d-b0dd-14eec6546bc8)

 and the second problem when enable RTL layout and when enable the sidebarCollapsibleOnDesktop in panel 
 the sidebar not show  
![111](https://github.com/filamentphp/filament/assets/7056487/f736ea85-9db4-445c-af8e-e102161a9844)

**Comment by danharrin** [comment]
Run `php artisan about`, what Filament version

**Reply by NazarAli** [reply]
filament v3.0.0

**Reply by NazarAli** [reply, marked as answer]
thank bro i updated to v3.0.40 and work fine
